src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -945,11 +889,50 @@ class ConsignmentRouteSpec extends TestContainerUtils with Matchers with TestReq
       parentRef: Option[Reference] = None
   ): UUID = {
     utils.createFile(fileId, consignmentId, NodeType.directoryTypeIdentifier, fileName, parentId = parentId, fileRef = fileRef, parentRef = parentRef)
-    staticMetadataProperties.foreach(p => utils.addFileMetadata(UUID.randomUUID().toString, fileId.toString, p.name, p.value))
+    addStaticMetaData(utils, fileId, false)
+    // staticMetadataProperties.foreach(p => utils.addFileMetadata(UUID.randomUUID().toString, fileId.toString, p.name, p.value))
     utils.addFileMetadata(UUID.randomUUID.toString, fileId.toString, ClientSideOriginalFilepath, fileName)
     fileId
   }
 
+  private def addStaticMetaData(utils: TestUtils, parentId: UUID, addProperty: Boolean): Unit = {
```

Comment:
I'd rename this function to `addDefaultMetadata` as these are set to a default value now from the DB. Also  replace `static` with `default` where its used within the function

The `static` is a hangover for when the property values were being statically held in the API code before they were added to the DB.